backend/danswer/db/tasks.py

```diff
@@ -15,7 +15,7 @@ def get_latest_task(
 ) -> TaskQueueState | None:
     stmt = (
         select(TaskQueueState)
-        .where(TaskQueueState.task_name == task_name)
+        .where(TaskQueueState.task_name.like(f"%{task_name}%"))
```

Comment:
why is this change needed? This feels like it should be a separate function (e.g. one for getting a task based on a specific name and another for getting the latest of a "family" of tasks)